source/episodes/004.markdown

```diff
@@ -1,20 +1,20 @@
 ---
 date: 2015-07-25
-excerpt: Can Mr Robot keep up the streak of good episodes? Or will it be haunted by its own daemons? Listen to find out!
+excerpt: Can Mr Robot keep up the streak of good episodes? Listen to find out!
```

Comment:
Total brain fart when I released this episode. The title of that episode of Mr Robot was `eps1.2_d3bug.mkv`, not `eps1.3_da3m0ns.mp4`. Fixed now.